src/test/run-pass/syntax-extension-source-utils-files/includeme.fragment

```diff
@@ -2,6 +2,6 @@
 
 {
     assert!(file!().ends_with("includeme.fragment"));
-    assert!(line!() == 5u);
+    assert!(line!() == 5_u32);
```

Comment:
FWIW, I personally don't dislike `5_u32`; it can make it easier to read by separating the actual number from the digits `32`.